src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -1036,64 +1036,195 @@ public void testPrintSuppressionOnMoreThanOneFile(@SysErr Capturable systemErr,
     public void testGenerateXpathSuppressionOptionOne(@SysErr Capturable systemErr,
             @SysOut Capturable systemOut) throws IOException {
         final String expected = addEndOfLine(
-            "<?xml version=\"1.0\" encoding=\"UTF-8\"?>",
+                "<?xml version=\"1.0\" encoding=\"UTF-8\"?>",
                 "<!DOCTYPE suppressions PUBLIC",
                 "    \"-//Checkstyle//DTD SuppressionXpathFilter Experimental Configuration 1.2"
                     + "//EN\"",
                 "    \"https://checkstyle.org/dtds/suppressions_1_2_xpath_experimental.dtd\">",
                 "<suppressions>",
                 "<suppress-xpath",
                 "       files=\"InputMainComplexityOverflow.java\"",
+                "       checks=\"IndentationCheck\"",
```

Comment:
too much changes .... we need to select different file with low level of violations.

Reply:
This Xpath is generated because this method follows `google_checks.xml` file as configuration.
Since indentation xpath generatiion is now possible, It is logging those xpaths. Should I add suppressions to this configuration or modify the input file provided here to little less complex code so that low code lines for indentation xpath is generated.

Reply:
My bad, since this xpath-generator is only for NpathComplexity Test, the generated x-path for indentation shouldn't be messed up with it. The test is testing this file against `google_checks.xml` configuration  [(Input File)](https://github.com/checkstyle/checkstyle/blob/master/src/test/resources/com/puppycrawl/tools/checkstyle/main/InputMainComplexityOverflow.java) which has indentation level 4 in every if block, which is conflicting with the `google_check` under indentation rule. So I am modifying this file to change indentation level to 2, so that no x-path suppression for indentation get generated.

Instead I am adding a new test to test indentation rules.